.github/workflows/build.yml

```diff
@@ -17,8 +17,8 @@ jobs:
         strategy:
             fail-fast: true
             matrix:
-                laravel: [^6, ^7, ^8]
-                php: [7.4, 8.0]
+                laravel: [^6, ^7, ^8, 9.0]
+                php: [7.4, 8.0, 8.0]
```

Comment:
Seems the PHP version got duplicated. We'll clean that up for next time.

Reply:
Thanks!